relnotes/windows-release-notes.md

```diff
@@ -1,19 +1,16 @@
 # Windows Release Notes
-This file contains the summary of changes in Windows OS releases, available through the Insider Canary Channel, for Win32 app isolation.
+This file contains the summary of newly supported features in Windows OS releases, available through the Insider Canary Channel, for Win32 app isolation.
 Release notes for the tools can be found in the [releases](https://github.com/microsoft/win32-app-isolation/releases) section of the repo.
 
 ## Windows build 25357 (2023-05-04)
-This Windows release contains the underlying infrastructure to install and run isolated applications, and support for capabilities for
-these applications which include:
 1. Implicit brokering for the open file dialog and other APIs.
 2. Printing.
 3. System tray icons.
 4. Shell notifications.
 5. File system privacy settings to reset file access permissions. 
 6. Integration with Windows Privacy App Permissions pages.
 
-## Coming soon 
-Windows releases on the Insider Canary Channel after June 2023 will add support for capabilities for isolated applications which include:
+## Windows build 25905 (2023-07-23)
```

Comment:
```suggestion
## Windows build 25905 (2023-07-12)
```

Reply:
An offset when I clicked on the keyboard. Did not realize that, thanks for catching it!